pkg/controllers/targetversion/target_version_controller.go

```diff
@@ -134,19 +101,6 @@ func (c *targetVersionController) sync(ctx context.Context, syncContext factory.
 	return common.UpdateControllerConditions(c.operatorClient, knownConditionNames, foundConditions)
 }
 
-func (c *targetVersionController) getRouteSecret() (*corev1.Secret, []operatorv1.OperatorCondition) {
-	routerSecret, err := c.secretLister.Secrets("openshift-authentication").Get("v4-0-config-system-router-certs")
-	if err != nil {
-		return nil, []operatorv1.OperatorCondition{{
-			Type:    "OAuthVersionRouteSecretDegraded",
```

Comment:
should we add this to the `staleConditions` list ?

Reply:
that's a good observation, yes